tests/ci/run_cryptofuzz.sh

```diff
@@ -67,12 +73,18 @@ SRC_CORPUS="$CRYPTOFUZZ_SEED_CORPUS"
 # 5 minutes for building AWS-LC and Cryptofuzz
 # 16 minutes (1000 seconds) of fuzzing
 # 24 minutes of cleanup and merging in new inputs
-TIME_FOR_EACH_FUZZ=1000
+TIME_FOR_EACH_FUZZ=${TIME_FOR_EACH_FUZZ:=1000}
 
 # Some fuzz tests can take a while but still pass. This is a tradeoff: less false positive noise, but some inputs that take
 # a long time could lead to a denial of service avenue. We're mostly interested in correctness and memory safety at this
 # time so we're willing to take the fit on fuzz speed
```

Comment:
minor nit: s/fit/hit/

Reply:
We'll need to get this fixed upstream and then pull it in. I'll cut an issue against the LC repo.

Reply:
hm, it's a pretty inconsequential typo. not sure if it's worth chasing down